boards/arm/ubx_bmd330eval_nrf52810/ubx_bmd330eval_nrf52810_defconfig

```diff
@@ -0,0 +1,34 @@
+# BMD-330-EVAL board configuration
+
+# Copyright (c) 2021 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52810_QFAA=y
+CONFIG_BOARD_UBX_BMD330EVAL_NRF52810=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# additional board options
+CONFIG_GPIO_AS_PINRESET=y
+
+# Bluetooth not enabled by default on BMD-330 due to RAM limitations when
+# running the default set of kernel tests.
+# Enable this in your prj.conf to include Bluetooth support
```

Comment:
This comment seems to be outdated (it's probably copied from nrf52dk_nrf52810_defconfig where it is outdated as well).

Reply:
Comments are removed - Yes, it was copied from nrf52dk_nrf52810